dali/operators/decoder/nvjpeg/nvjpeg_memory.cc

```diff
@@ -87,6 +88,7 @@ struct NVJpegMem {
 
   static NVJpegMem &instance() {
     // ensure proper destruction order
+    (void)mm::GetDefaultResource<mm::memory_kind::host>();
```

Comment:
Side question - these resources still use the global, common pool but provides separate stats regarding allocations, right?